### PR TITLE
site: translation of home and footer

### PR DESCRIPTION
**Pull request by benjycui**
Close: #1471

### CODE REVIEW

**Comment by benjycui** [comment]
@visvadw I am not sure if you are interested in this.

File: site/theme/en-US.js [hunk 1]
```diff
@@ -11,5 +11,33 @@ module.exports = {
     'app.header.menu.spec': 'Specification',
     'app.header.menu.resource': 'Resource',
     'app.header.lang': '中文',
+    'app.home.slogan': 'A design language for UI',
```
Comment (afc163) [inline site/theme/en-US.js]: One UI design language

Comment (benjycui) [inline site/theme/en-US.js]: 😂 感觉不对啊。

File: site/theme/en-US.js [hunk 2]
```diff
@@ -11,5 +11,33 @@ module.exports = {
...
+    'app.home.start': 'Get Started',
```
Comment (visvadw) [inline site/theme/en-US.js]: 'app.home.start': 'Getting Started',

File: site/theme/en-US.js [hunk 3]
```diff
@@ -11,5 +11,33 @@ module.exports = {
...
+    'app.home.best-practice': 'Best Practice',
+    'app.home.experience': 'Designed by experienced team, and showcase dozens of inspiring projects.',
+    'app.home.design-pattern': 'Design Pattern',
+    'app.home.pattern': 'Provide solution for usual problems which we may meet in developing enterprise-like complex UI.',
```
Comment (visvadw) [inline site/theme/en-US.js]: 'app.home.pattern': 'Provide solutions for usual problems that may be encountered while developing enterprise-like complex UIs.',

File: site/theme/en-US.js [hunk 4]
```diff
@@ -11,5 +11,33 @@ module.exports = {
...
+    'app.home.reusable-components': 'Dozens of Reusable Components',
+    'app.home.components-intro': 'Dozens of flexible and practical reusable components, which will power productivity up.',
```
Comment (visvadw) [inline site/theme/en-US.js]: 'app.home.components-intro': 'Dozens of flexible and practical reusable components that increase your productivity.',

File: site/theme/en-US.js [hunk 5]
```diff
@@ -11,5 +11,33 @@ module.exports = {
...
+    'app.home.learn-more': 'Learn more',
+    'app.home.sub-slogan': 'A Little Happiness in Hand',
+    'app.home.vision': 'This is a design language which dedicate to improve user experience and design experience.',
```
Comment (visvadw) [inline site/theme/en-US.js]: 'app.home.vision': 'This is a design language dedicated to improve the user and design experience.',

**Comment by visvadw** [comment]
@benjycui Done reviewing it. 👍

**Comment by benjycui** [comment]
Thanks and updated!

**Comment by kvetoslavnovak** [comment]
"The Design Language for UI"

**Comment by benjycui** [comment]
@kvetoslavnovak updated, thanks!

**Comment by afc163** [comment]
一个用 `One` 更好。

**Comment by benjycui** [comment]
Updated.

**Comment by kvetoslavnovak** [comment]
Then I believe it should be "The One Design Language for UI"

File: site/theme/en-US.js [hunk 6]
```diff
@@ -11,5 +11,33 @@ module.exports = {
...
+    'app.footer.dev-tools': 'Developer Tools',
+    'app.footer.dva': 'Framework',
+    'app.footer.links': 'Links',
+    'app.footer.mobile': 'Mobile',
+    'app.footer.data-vis': 'Data Visualization',
+    'app.footer.data-vis-spec': 'Specification of Data Visualization',
+    'app.footer.motion': 'Motion',
+    'app.footer.material': 'Design Materials',
+    'app.footer.community': 'Community',
+    'app.footer.change-log': 'Change Log',
```
Comment (afc163) [inline site/theme/en-US.js]: Changelog ?

File: site/theme/en-US.js [hunk 7]
```diff
@@ -11,5 +11,33 @@ module.exports = {
...
+    'app.footer.repo': 'Repository',
+    'app.footer.scaffold': 'Scaffold',
+    'app.footer.dev-tools': 'Developer Tools',
+    'app.footer.dva': 'Framework',
+    'app.footer.links': 'Links',
+    'app.footer.mobile': 'Mobile',
+    'app.footer.data-vis': 'Data Visualization',
+    'app.footer.data-vis-spec': 'Specification of Data Visualization',
+    'app.footer.motion': 'Motion',
+    'app.footer.material': 'Design Materials',
```
Comment (afc163) [inline site/theme/en-US.js]: Design Resource

File: site/theme/en-US.js (hunk 6 above)
Comment (benjycui) [inline site/theme/en-US.js]: Change Log 是分开的。

**Pull request merged**